Sources/Data/models/Model.xcdatamodeld/Model17.xcdatamodel/contents

```diff
@@ -60,6 +60,11 @@
             </uniquenessConstraint>
         </uniquenessConstraints>
     </entity>
+    <entity name="CustomFilterListSetting" representedClassName="Data.CustomFilterListSetting" syncable="YES">
+        <attribute name="externalURL" optional="YES" attributeType="URI"/>
+        <attribute name="isEnabled" optional="YES" attributeType="Boolean" usesScalarValueType="YES"/>
+        <attribute name="uuid" optional="YES" attributeType="String"/>
+    </entity>
```

Comment:
Do we want a new model version for this?

Reply:
I forgot what the rule is here. I'm adding a new entry so I thought its not necessary but I can add one if its best

Reply:
Done